test/Analysis/test-allocation.mlir

```diff
@@ -628,7 +628,8 @@ tt.func @scan_alloc(%x : tensor<8x16xf32, #AL>) {
 }
 
 // expected-remark @below {{warp_specialize_default_region}}
-// expected-remark @below {{size = 32}}
+// expected-remark @below {{size = 33}}
```

Comment:
why did the size change here? There are no value captured?

Reply:
A little bit of shared memory is used to communicate the partition states.